src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Controllers/ApprenticeshipUpdateController.cs

```diff
@@ -23,17 +25,10 @@ public ApprenticeshipUpdateController(IMediator mediator, IModelMapper modelMapp
         }
 
         [HttpGet]
-        [Route("pending")]
-        public async Task<IActionResult> GetPendingUpdate(long apprenticeshipId)
+        public async Task<IActionResult> GetApprenticeshipUpdates(long apprenticeshipId,[FromQuery] ApprenticeshipUpdateStatus? status)
```

Comment:
Shouldn't the Api request type be the parameter here, instead of these primitives?